x-pack/plugins/ml/public/embeddables/anomaly_swimlane/embeddable_component.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import type { CoreStart } from '@kbn/core/public';
+import {
+  EmbeddableFactory,
+  EmbeddableInput,
+  EmbeddableOutput,
+  EmbeddableRoot,
+  useEmbeddableFactory,
+} from '@kbn/embeddable-plugin/public';
+import { EuiLoadingChart } from '@elastic/eui';
+import { ANOMALY_SWIMLANE_EMBEDDABLE_TYPE, AnomalySwimlaneEmbeddableInput } from '../..';
+import type { MlStartDependencies } from '../../plugin';
+
+export function getEmbeddableComponent(core: CoreStart, plugins: MlStartDependencies) {
+  const { embeddable: embeddableStart } = plugins;
+  const factory = embeddableStart.getEmbeddableFactory(ANOMALY_SWIMLANE_EMBEDDABLE_TYPE)!;
```

Comment:
is it definitely safe to assert that this is not `undefined`?

Reply:
yes, because we register a case attachment type after embeddables